consul/agents.go

```diff
@@ -10,20 +10,21 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/allegro/marathon-consul/metrics"
 	"github.com/allegro/marathon-consul/utils"
-	consulapi "github.com/hashicorp/consul/api"
 )
 
 type Agents interface {
-	GetAgent(agentAddress string) (agent *consulapi.Client, err error)
+	GetAgent(agentAddress string) (agent *Agent, err error)
```

Comment:
:+1:  For making `Agents` interface more consitent but please do it in a separate PR.